src/xrGame/Weapon.cpp

```diff
@@ -1309,6 +1310,10 @@ void CWeapon::UpdateHUDAddonsVisibility()
 
     //.	return;
 
+    u16 bone_id = HudItemData()->m_model->LL_BoneID(wpn_grenade_launcher);
+    if (bone_id == BI_NONE)
+        wpn_grenade_launcher = WPN_GRENADE_LAUNCHER_SOC;
```

Comment:
Забыл здесь строку тоже поправить
```suggestion
        bone_id = pWeaponVisual->LL_BoneID(wpn_grenade_launcher_soc);
```